db/migrate/20210107112850_add_patient_id_index_to_materialized_patient_summary.rb

```diff
@@ -0,0 +1,7 @@
+class AddPatientIdIndexToMaterializedPatientSummary < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    add_index :materialized_patient_summaries, :id, algorithm: :concurrently
```

Comment:
This is very important for making the view useful, otherwise it'll continue to be as slow. The `id` here is the patient id.